apps/backend/src/lib/schema/logistics/on_cancel.ts

```diff
@@ -402,8 +420,12 @@ export const onCancelSchema = {
 															},
 															required: ["start", "end"],
 														},
+														timestamp:{
+															type: "string",
+															formate: "date-time",
+														}
 													},
-													required: ["range"],
+													required: ["range", "timestamp"],
```

Comment:
timestamp is mandatory only at specific fulfillment states. reason for adding this?